dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1481,6 +1478,7 @@ def test_reductions_frame_dtypes_numeric_only(func):
         getattr(ddf, func)(**kwargs),
     )
     if not DASK_EXPR_ENABLED:
+        # This actually works in dask-expr
```

Comment:
I don't get this?

Reply:
This doesn't raise and we have tests for this over in dask-expr